src/mail/view/MailViewer.ts

```diff
@@ -358,6 +358,10 @@ export class MailViewer implements Component<MailViewerAttrs> {
 						event.preventDefault()
 					}
 				},
+				onkeydown: (event: Event) => {
```

Comment:
Should this be added to root node inside the shadow dom instead?

Reply:
That might be better.

I went and updated it.